cmd/deployment.go

```diff
@@ -15,16 +15,25 @@ import (
 	"github.com/spf13/cobra"
 )
 
+const (
+	celeryExecutorArg     = "celery"
+	localExecutorArg      = "local"
+	kubernetesExecutorArg = "kubernetes"
+	k8sExecutorArg        = "k8s"
+)
+
 var (
 	errUpdateDeploymentInvalidArgs = errors.New("must specify a deployment ID and at least one attribute to update")
 	errServiceAccountNotPresent    = errors.New("must provide a service-account label with the --label (-l) flag")
 )
 
 var (
-	allDeployments        bool
-	cancel                bool
-	hardDelete            bool
-	executor              string
+	allDeployments bool
+	cancel         bool
+	hardDelete     bool
+	executor       string
+	// have to use two different executor flags for create and update commands otherwhise both commands override this value
+	executorUpdate        string
```

Comment:
from this PR changes, I didn't get why we need two separate executor flags for create & update deployment because in any given execution only one of the two commands would be executed 🤔 , can you explain further.

Reply:
I tested this change locally and I got into an error where `executor` was `""` (empty string).

That's because in both `update` and `create` we write the line:
```go
cmd.Flags().StringVarP(&executor, "executor", "e", "", "Add executor parameter: local, celery, or kubernetes")
```
This flag --executor is handled 2 times, and the declaration in the "update" method overrides the one in create deployment, which means the --executor flag will ALWAYS default to empty string.

If I change it to default to "celery" in both places, the update method will always have the `executor` variable populated with the default value "celery", even when the user does not want to update his deployment's executor configuration. This would lead to unwanted side-effects, so the best solution to avoid bugs is to decouple the executor variable for both commands.